docs/02-sygma-sdk/05-Examples/01-NodeJS-Token-Transfer-Example.md

```diff
@@ -0,0 +1,342 @@
+---
+slug: /sdk/examples/node-js-example
+id:  quickstart-nodejs-example
+title: Node.js Token Transfer Example
+description: Section that describes how to transfer token using nodeJS.
+sidebar_position: 1
+draft: false
+---
+
+:::info 
+In the following example, we will setup and demonstrate a cross-chain ERC-20 token transfer between the same address from Goerli to Sepolia. This will be shown through the command line in a backend Node.js environment using the Sygma SDK.
+:::
+
+#### Prerequisites
+
+- Setup [Node.js](https://nodejs.org/en/about)(version 16 or above) for the backend JavaScript runtime environment.
```

Comment:
I will add here install Node.js or have node.js installed. We don't need to referecen `npm`. Once you install node, npm comes with it

Reply:
updated and removed the bullet for npm and yarn